qutebrowser/mainwindow/tabwidget.py

```diff
@@ -451,11 +483,33 @@ def tabSizeHint(self, index):
             # get scroll buttons as soon as needed.
             size = minimum_size
         else:
+            tab_width_pinned_conf = config.get('tabs', 'pinned-width')
+
+            try:
+                pinned = self.tab_data(index, 'pinned')
+            except KeyError:
+                pass
+            else:
```

Comment:
I think this would be slightly more readable when just setting `pinned = False` in the `except` block and not having an `else`.